applications/luci-app-csshnpd/htdocs/luci-static/resources/view/sshnpd/config.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+'require view';
+'require form';
+
+// Project code format is tabs, not spaces
+return view.extend({
+	render: function() {
+		var m, s, o;
+
+		/*
+		The first argument to form.Map() maps to the configuration file available
+		via uci at /etc/config/. In this case, 'sshnpd' maps to /etc/config/sshnpd.
+
+		If the file is completely empty, the form sections will indicate that the
+		section contains no values yet. As such, your package installation (LuCI app
+		or software that the app configures) should lay down a basic configuration
+		file with all the needed sections.
+
+		The relevant ACL path for reading a configuration with UCI this way is
+		read > uci > ["example"]
+
+		The relevant ACL path for writing back the configuration is
+		write > uci > ["example"]
+		*/
+		m = new form.Map('sshnpd', _('NoPorts'),
+			_('Daemon Configuration'));
+
+		s = m.section(form.TypedSection, 'sshnpd', _('sshnpd config'));
+		s.anonymous = true;
+
+		s.option(form.Value, 'atsign', _('Device atSign'),
```

Comment:
What about input validation?
It would also be nice if you could add a short description of what this option is for.